modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -295,14 +307,28 @@ endif::restricted[]
 ifndef::openshift-origin[]
 <13> The SSH public key for the `core` user in {op-system-first}.
 endif::openshift-origin[]
-ifdef::openshift-origin[]
-<12> The SSH public key for the `core` user in {op-system-first}.
-endif::openshift-origin[]
 +
 [NOTE]
 ====
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
+ifndef::openshift-origin[]
+<14> Activate a predefined set of cluster capabilities.
+<15> Add the additional cluster capabilities you want to enable here.
++
+[NOTE]
+====
+The following sets of cluster capabilities are predefined:
+
+`None`:: All capabilities are disabled.
+`v4.11`:: Includes `baremetal`, `marketplace`, and `openshift-samples`.
+`v4.12`:: Includes all capabilities from `v4.11`, and `Console`, `Insights`, `Storage`, `CSISnapshot`.
+`vCurrent`:: All available capabilities are enabled.
```

Comment:
It seems like it's going to be some maintenance work to try and keep multiple copies of this information up to date as we extend the number of caps an sets in future 4.y.  I'm not docs, so :shrug:, no worries from me if this is the expected way to do things.  But if it was me, I'd prefer an approach that centralized management, so we didn't have to remember to bump multiple locations when the enums changed.